developer-hub/src/polkadot_sdk/polkadot.rs

```diff
@@ -0,0 +1,81 @@
+//! # Polkadot
+//!
+//! Implementation of the Polkadot node/host in Rust.
+//!
+//! ## Getting Involved
+//!
+//! - [Polkadot Forum](https://forum.polkadot.network/)
+//! - [Polkadot Parachains](https://parachains.info/)
```

Comment:
Is this a way to get involved? Maybe change the title to "Getting involved/learn more"